src/OmegaPoker.t.sol

```diff
@@ -192,6 +193,20 @@ contract OmegaPokerTest is DSTest {
         assertEq(omegaPoker.osmCount(), osmcount);    // No osm to remove
     }
 
+    function testRefreshZeroPip() public {
+        // grant ourselves authority on the ilk registry
+        address registry = address(omegaPoker.registry());
+        hevm.store(registry, keccak256(abi.encode(address(this), uint(0))), bytes32(uint(1)));
+
+        // Ensure we can refresh
+        omegaPoker.refresh();
+
+        RegistryEdit(registry).file("ETH-A", "pip", address(0)); // Remove PIP
```

Comment:
Probably want to add a require after this just to ensure registry.pip("ETH-A") == address(0)

Reply:
https://github.com/makerdao/megapoker/pull/38/commits/fbe734bf5a47af87bde28058492f65e5690cfd69